src/OpenTelemetry/Metrics/Reader/MetricReaderExt.cs

```diff
@@ -52,7 +52,15 @@ internal virtual List<Metric> AddMetricWithNoViews(Instrument instrument)
         Debug.Assert(instrument != null, "instrument was null");
         Debug.Assert(this.metrics != null, "this.metrics was null");
 
-        var metricStreamIdentity = new MetricStreamIdentity(instrument!, metricStreamConfiguration: null);
+        MetricStreamConfiguration? metricStreamConfiguration = null;
+
+        // Apply default histogram aggregation if configured
+        if (this.DefaultHistogramAggregation.HasValue)
+        {
+            metricStreamConfiguration = this.CreateDefaultHistogramConfiguration(instrument);
+        }
```

Comment:
This seems weird to me to just test for `HasValue` but not use it explicitly.

I think it would be better passed to the `CreateDefaultHistogramConfiguration()` method like:

```csharp
if (this.DefaultHistogramAggregation is {} aggregation)
{
    metricStreamConfiguration = this.CreateDefaultHistogramConfiguration(instrument, aggregation);
}
```